ui/pages/swaps/searchable-item-list/item-list/item-list.component.js

```diff
@@ -107,7 +108,10 @@ export default function ItemList({
               <div className="searchable-item-list__labels">
                 <div className="searchable-item-list__item-labels">
                   {primaryLabel ? (
-                    <span className="searchable-item-list__primary-label">
+                    <span
+                      className="searchable-item-list__primary-label"
+                      data-testid="searchable-item-list-primary-label"
+                    >
```

Comment:
Do we want use `Text` instead of `span` here @georgewrmarshall

Reply:
This PR only adds `data-testid` attributes in this existing file. We have a user story to refactor a few remaining components in Swaps to use the new DS components (we use them in most of the redesigned Swaps already), so I would prefer do it in another PR.